jest.config.json

```diff
@@ -8,6 +8,14 @@
 	"coveragePathIgnorePatterns": [
 		"/node_modules/"
 	],
+	"coverageThreshold": {
+		"./packages/**/*.js": {
+			"branches": 100,
+			"functions": 100,
+			"lines": 100,
+			"statements": 100
```

Comment:
I decided what the hell, let's be ambitious and fail the tests if they don't have 100% coverage 😂  if we don't do it from the start then it'll be nearly impossible to do it later